lib/ui/creators_corner_page.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
```

Comment:
Later on for us to keep in mind but if we import more duplicate packages for all the pages, might be nice to define a parent file & just reference that!